og_access/og_access.api.php

```diff
@@ -14,23 +14,23 @@
 /**
  * Allow modules to mark group privacy change.
  *
- * @param &$context
- *   Array including 3 keys:
- *   'entity' - the submitted group entity.
- *   'entity_type' - String. The type of the entity that was submitted.
- *   'change_detected' - Boolean. Marks whether the group privacy changed or not.
- *                       FALSE by default, and changeable be each hook
- *                       implementation.
+ * @param array $context
+ *   Array with the following keys:
+ *   'entity': the group entity.
+ *   'entity_type':  The group type.
  */
-function hook_og_access_invoke_node_access_acquire_grants_alter(&$context) {
+function hook_og_access_invoke_node_access_acquire_grants($context) {
```

Comment:
you need to also document the ``hook_og_access_invoke_node_access_acquire_grants_alter()``